app-config.yaml

```diff
@@ -67,6 +67,14 @@ proxy:
 # '/test':
 #   target: 'https://example.com'
 #   changeOrigin: true
+  endpoints:
+    '/github-api':
```

Comment:
Is this needed for the roadie API step? If not, what's it needed for?

Reply:
@sheaphillips - yes it's for the roadie API step. In the [scaffolder-backend-module-http-request documentation](https://www.npmjs.com/package/@roadiehq/scaffolder-backend-module-http-request), under the "Supported methods" section it says:

>The path should always point to a proxy entry with the following format: `proxy/<proxy-path>/<external-api-path>`

I found out the hard way that the leading slash is important.